miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/SampleStockDto.java

```diff
@@ -10,8 +10,8 @@
 public class SampleStockDto extends SampleTissueDto {
 
   private Long tissueProcessingClassId;
-  private String concentration;
```

Comment:
Was `concentration` intended to be removed here?

Reply:
Yes. It's already on `SampleDto` and the override here was doing nothing